crt_portal/cts_forms/views.py

```diff
@@ -517,7 +517,7 @@ def get(self, request):
         ids = request.GET.getlist('id')
         ids_count = len(ids)
 
-        bulk_actions_form = BulkActions()
+        bulk_actions_form = BulkActionsForm(requested_query)
```

Comment:
It looks like we need to handle this in the same way we are in `post` below. This takes the full query, regardless of selected Reports, and uses it to generate initial values. 

Resulting in `MULTIPLE` values being rendered on the form for `status` when 3 reports with an identical `status` are selected.

Reply:
Great catch! Thanks. Tossed up a fix for this.